tests/unit/layouts/ModelizerDrawLayout.spec.js

```diff
@@ -134,13 +134,25 @@ describe('Test page component: ModelizerDrawLayout', () => {
     });
 
     it('should do nothing when there is no plugin', async () => {
-      PluginManager.getPluginByName = jest.fn(() => null);
+      const notifySpy = jest.spyOn(Notify, 'create');
 
-      Notify.create = jest.fn();
+      await wrapper.vm.initView();
 
+      expect(notifySpy).not.toHaveBeenCalled();
+    });
+  });
+
+  describe('Test function: arrangeComponentsPosition', () => {
+    it('should arrange a component position and redraw on click', async () => {
+      // We need this to load the plugin.
```

Comment:
`initView()` is already done in `onMounted`, so not necessary here

Reply:
deleted